charts/featureform/charts/metadata/templates/deployment.yaml

```diff
@@ -20,6 +20,8 @@ spec:
         chart: featureform
         run: featureform-metadata-server
         app: featureform-metadata-server
+      annotations:
```

Comment:
What do these do?

Reply:
these are a custom timestamp that change the resulting deployment yaml spec when we call `helm upgrade`. 
if not, when we call `helm upgrade` if k8s sees no diff in the resulting spec it won't cause the deploys to restart (even if technically you posted a new image with the same tag in docker).